### PR TITLE
docs: Add Troubleshooting Debugging

### PR DESCRIPTION
**Pull request by eriklimakc**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/debugging-mobile.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+uid: Uno.Development.DebuggingOnMobile
+---
+
+# Debugging on Mobile
```
Comment (Youssef1313) [inline doc/articles/features/debugging-mobile.md:5]: This applies to Wasm as well? Basically, non-Desktop?

Comment (morning4coffe-dev) [reply doc/articles/features/debugging-mobile.md:5]: Good point. We should update it, maybe connect it with the Wasm specific docs? "Debugging on Uno targets"

Comment (jeromelaban) [reply doc/articles/features/debugging-mobile.md:5]: This is for all mono targets. It looks more like a troubleshooting guide, let's call it Debugging troubleshooting.

**Review by morning4coffe-dev — COMMENTED** [review]
Just a detail, but I think we should change the name of the file from debugging-mobile.md as well

**Review by morning4coffe-dev — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

**Comment by eriklimakc** [comment]
@Mergifyio backport release/stable/5.4

**Pull request merged**